nautobot-app/{{ cookiecutter.project_slug }}/tasks.py

```diff
@@ -159,17 +159,17 @@ def run_command(context, command, **kwargs):
         # Check if nautobot is running, no need to start another nautobot container to run a command
         docker_compose_status = "ps --services --filter status=running"
         results = docker_compose(context, docker_compose_status, hide="out")
-        if "nautobot" in results.stdout:
-            compose_command = "exec"
-        else:
-            compose_command = "run --rm --entrypoint=''"
 
+        command_env_args = ""
         if "command_env" in kwargs:
             command_env = kwargs.pop("command_env")
             for key, value in command_env.items():
-                compose_command += f' --env="{key}={value}"'
+                command_env_args += f' --env="{key}={value}"'
 
-        compose_command += f" -- nautobot {command}"
+        if "nautobot" in results.stdout:
+            compose_command = f"exec{command_env_args} nautobot {command}"
+        else:
+            compose_command = f"run{command_env_args} --rm --entrypoint='{command}' nautobot"
```

Comment:
This fixes the issue where the entrypoint script still runs every time you run a command in a container (`invoke ruff` runs `nautobot-server post_upgrade` before running ruff)